src/core/Paperwork/US_TN/FormCR3947Rev0518.tsx

```diff
@@ -0,0 +1,743 @@
+                Driver’s License #
+              </Item>
+              <Input
+                placeholder="DL #"
+                defaultValue={form.driversLicense}
+                id="drivers_license"
+              />
+              <Grid rows="1fr 1fr">
+                <Item>
+                  {/* A checkbox input is nested below  */}
+                  {/* eslint-disable-next-line jsx-a11y/label-has-for */}
+                  <label>
+                    <Checkbox
+                      defaultChecked={form.driversLicenseSuspended === "1"}
+                    />{" "}
+                    Suspended
+                  </label>
+                </Item>
+                <Item>
+                  {/* A checkbox input is nested below  */}
+                  {/* eslint-disable-next-line jsx-a11y/label-has-for */}
+                  <label>
+                    <Checkbox
+                      defaultChecked={form.driversLicenseRevoked === "1"}
+                    />{" "}
+                    Revoked
+                  </label>
+                </Item>
+              </Grid>
+              <Item center>
+                <Item as="label">
+                  for <Input placeholder="Number." /> years.
+                </Item>
+              </Item>
+            </Grid>
+            <Grid columns="100px 1fr 30px 1fr">
+              <Item as="label" htmlFor="county">
+                County of Conviction
+              </Item>
+              <Input
+                placeholder="County."
+                defaultValue={form.convictionCounty}
+                id="county"
+              />
+              <Item as="label" htmlFor="court_name">
+                Court
+              </Item>
+              <Input
+                placeholder="Court Name."
+                defaultValue={form.courtName}
+                id="court_name"
+              />
+            </Grid>
+            <Grid columns="100px 1fr">
+              <Item as="label" center>
+                <span>
+                  Docket #{" "}
+                  <Emphasize>
+                    <strong>List all </strong>
+                  </Emphasize>
+                </span>
+              </Item>
+              <Input placeholder="Docket(s)." defaultValue={form.allDockets} />
+            </Grid>
+            <Grid columns="100px 1fr">
+              <Item>
+                Conviction Offenses
+                <br />
+                <Item
+                  center={false}
+                  as="label"
+                  style={{ whiteSpace: "normal" }}
+                >
+                  <Checkbox />
+                  <Emphasize as="strong">
+                    See additional offenses on reverse side.
+                  </Emphasize>
+                </Item>
+              </Item>
+              <Grid rows="repeat(5, 1fr)">
+                <Input
+                  placeholder="Offenses."
+                  defaultValue={form.offenseType && form.offenseType[0]}
```

Comment:
You can use the optional chaining operator (`?.`) with array access as well:
```suggestion
                  defaultValue={form.offenseType?.[0]}
```

Reply:
:+1: I was trying this out but had no idea about the syntax.